src/components/Button.js

```diff
@@ -112,6 +113,26 @@ const StyledButton = styled.a`
     :focus {
       box-shadow: 0 0 0 .2rem ${hexToRgba(p.theme.colors.primary, 0.5)};
     }`}
+    ${p =>
+    p.color !== 'primary' &&
+    p.color !== 'secondary' &&
+    p.color !== 'tertiary' && // some color other than the variants
```

Comment:
Is this to change the default button color? Would this work as well?
```suggestion
    !p.color
```

Reply:
This is to change the button colour to a specified `color` (line 121) that is not one of the defined colours